.github/workflows/build_and_test.yml

```diff
@@ -170,7 +170,7 @@ jobs:
         pytest integration/local_emulator_multithreading_test.py
 
   qa_checks:
-    name: Run backend tests with QA endpoint
+    name: Run backend tests with QA and prod endpoint
```

Comment:
I'm not sure we should be running Prod tests on PRs. If necessary to split we could move Prod tests into another file and have it run on a schedule?

Reply:
I think it is very helpful to test both, I have split this up in the workflow and will only run the prod checks once per week and on request.